.github/workflows/ci-cd-pull-request.yml

```diff
@@ -6,7 +6,55 @@ env:
   BASE_IMAGE: ghcr.io/graphprotocol/graph-docs-staging
 
 jobs:
-  build:
+  lint:
```

Comment:
This is currently failing on this PR, not sure why. 🤔

Reply:
@saihaj – No idea why I had to make [these ugly changes](https://github.com/graphprotocol/docs/pull/661/files#diff-056cc9f48e6ecaeee1c02b3bea2b98c1b11453caa18b5f5254f1247cced34c8a) to make `pnpm lint` pass in CI (it works without those changes locally).